cmd/commento/main.go

```diff
@@ -17,6 +17,10 @@ func main() {
 		Die(err)
 	}
 	
+	if err := godotenv.Load(); err != nil {
```

Comment:
We should remove the lines below

```go
if fromEnv := os.Getenv("PORT"); fromEnv != "" {
	port = ":" + fromEnv
} else {
	port = ":8080"
}
```

We should add an `.env.sample` file with initial values, _sans_ secrets of course.